automation/src/test/scala/org/broadinstitute/dsde/firecloud/config/UserPool.scala

```diff
@@ -69,4 +69,14 @@ object UserPool {
     Temps.getRandomCredentials(n)
   }
 
+    /**
+    * Chooses a notebooksWhitelisted user.
+    */
+  def chooseNotebooksWhitelisted: Credentials = chooseNotebooksWhitelisteds(1).head
+
+  def chooseNotebooksWhitelisteds(n: Int): Seq[Credentials] = {
+    NotebooksWhitelisted.getRandomCredentials(n)
```

Comment:
This is great! Can we get more users in the whitelist (which currently seems to just be 1 user) to make sure that we don't overuse one user?